tests/conftest.py

```diff
@@ -65,6 +65,7 @@ def config():
         {"name": "jane", "nick": "jane", "bmo_id": 0},
         {"name": "jill", "nick": "jill", "bmo_id": 1},
         {"name": "otis", "nick": "otis", "bmo_id": 2},
+        {"name": "angel", "nick": "angel", "bmo_id": 4},
```

Comment:
Why did we skip 3 as a bmo id?

Reply:
That's one that's added outside of this fixture in some tests. Might as well be 10 or 50, and all these numbers may as well be random integers (in fact, may be better if that's the case, but that's out of scope for this PR).